{"name": "test_multiple_login_page_success[invalid_user2-wrong_password]", "status": "passed", "parameters": [{"name": "username", "value": "'invalid_user2'"}, {"name": "password", "value": "'wrong_password'"}], "start": 1753503580229, "stop": 1753503582476, "uuid": "7ba1f125-6e85-49ce-a0d1-29185e74425f", "historyId": "bff793983932d93056520584ccf3f372", "testCaseId": "90d4f72dd78db4ae114ce1f1cc9587a0", "fullName": "tests.test_multiple_login#test_multiple_login_page_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_multiple_login"}, {"name": "host", "value": "ahmads-MacBook-Pro.local"}, {"name": "thread", "value": "59772-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_multiple_login"}], "titlePath": ["tests", "test_multiple_login.py"]}